src/core/core-crypto.c

```diff
@@ -23,6 +23,10 @@
 #include "config.h"
 #endif
 
+#if defined(__APPLE__)
```

Comment:
should I move it above config.h include?

Reply:
I think it could be moved below, after all includes.
No other choice than defining this on macOS?

Reply:
did that:

```
#ifdef htonll
#define htobe64 htonll
#endif
```